### PR TITLE
Add AUR to the README

### PR DESCRIPTION
**Pull request by TabulateJarl8**
There wasn't a feature branch so I'm doing this on master

### CODE REVIEW

**Comment by Nytelife26** [comment]
I would strongly suggest solely linking to the AUR and letting users figure out how best to install from it for their system on their own. This varies for many systems and there are no one size fits all instructions.

**Comment by TabulateJarl8** [comment]
I'm open to that as well, I'm just not too sure where I should put things and how to format it, so I'll change it if you have any good ideas about that

**Comment by Nytelife26** [comment]
> I'm open to that as well, I'm just not too sure where I should put
> things and how to format it, so I'll change it if you have any good
> ideas about that

I would suggest something along the lines of

```md
Or, Arch Users may install [ugit via the AUR][aur-ugit]

[aur-ugit]: https://aur.archlinux.org/packages/ugit
```

File: README.md [hunk 1]
```diff
@@ -69,6 +69,13 @@
    # Mac
    curl -fsSL https://github.com/Bhupesh-V/ugit/releases/latest/download/ugit -o ugit && chmod +x ugit && mv ugit /usr/local/bin
    ```
+   
+   or Arch Linux users can install ugit via the AUR
```
Comment (Bhupesh-V) [inline README.md:73]:
```suggestion
   Or Arch Linux users can install [**ugit via AUR**](https://aur.archlinux.org/packages/ugit)
```

File: README.md [hunk 2]
```diff
@@ -69,6 +69,13 @@
...
+   
+   ```bash
+   git clone https://aur.archlinux.org/ugit.git && cd ugit
+   makepkg -si
+   ```
```
Comment (Bhupesh-V) [inline README.md:78]: No need

**Review by Bhupesh-V — CHANGES_REQUESTED** [review]
First of thanks a lot for checking out ugit and packaging it for AUR
I guess as @Nytelife26 pointed out linking the AUR would be just fine

Again, appreciate the effort 🙌🏽

**Comment by TabulateJarl8** [comment]
Sorry that I couldn't change it sooner, I had a power outage yesterday. It should be fixed now though

**Review by Bhupesh-V — APPROVED** [review]

**Comment by Bhupesh-V** [comment]
> Sorry that I couldn't change it sooner, I had a power outage yesterday. It should be fixed now though

Thanks again for the contrib, 
Take care 😷 🛡️

**Comment by Bhupesh-V** [comment]
@all-contributors please add @TabulateJarl8 for code and platform

**Pull request merged**